imports/plugins/core/orders/client/templates/orders.js

```diff
@@ -89,22 +79,35 @@ const OrderHelper =  {
 
 Template.orders.onCreated(function () {
   this.state = new ReactiveDict();
+  this.newLimit = new ReactiveVar(2);
```

Comment:
Could we combine these into one dictionary so you could use the filter as the key? e.g. orderLimit["new]]. That would help get around having to keep doing there different if/switch branches

`const orders = Orders.find(query, { limit: orderLimit[filter] }).fetch();`